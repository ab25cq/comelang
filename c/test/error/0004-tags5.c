/*
PATTERN: incomplete
PATTERN: c:7:
*/

struct X;
struct X { struct X x; };
