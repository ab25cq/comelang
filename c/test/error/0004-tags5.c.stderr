incomplete type inside struct/union at test/error/0004-tags5.c:7:21
struct X { struct X x; };
                    ^
